cnames_active.js

```diff
@@ -254,6 +254,7 @@ var cnames_active = {
   "avner": "avnerus.github.io", // noCF? (don´t add this in a new PR)
   "avo": "benjaminbhollon.github.io/avo-js",
   "avstream": "akash-joshi.github.io/avstream.js",
+  "awardify": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "awardify": "cname.vercel-dns.com", // noCF
```
As the CI error notes, you'll want the Cloudflare proxy disabled for Vercel